python/ctsm/test/test_unit_gen_mksurfdata_jobscript_single.py

```diff
@@ -136,7 +136,7 @@ def test_simple_derecho_args(self):
     def test_derecho_mpirun(self):
         """
         test derecho mpirun. This would've helped caught a problem we ran into
-        It will also be helpful when externals are updated to guide to solutions
+        It will also be helpful when sumodules are updated to guide to solutions
```

Comment:
scrolled through this and was thinking if you wanted to write submodules..

Reply:
Yep, the new use of git-fleximod uses the term submodules for externals rather than externals that we used in manage_externals. So this is correct. This only showed up in the PR for a little bit because it was showing changes from other PR's until it was fully updated to master.